### PR TITLE
feat: owner of source contract able to deploy contract

### PR DESCRIPTION
**Pull request by cuonghx-dev**

### CODE REVIEW

File: contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol [hunk 1]
```diff
@@ -237,7 +238,7 @@ abstract contract BasicNFTOmnibridge is
 
         bytes memory data = _prepareMessage(_token, _receiver, _tokenIds, _values);
 
-        bytes32 _messageId = _passMessage(data, _isOracleDrivenLaneAllowed(_token, _from, _receiver));
```
Comment (kazkondo) [inline contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol:240]: Please explain this

Comment (cuonghx-dev) [reply contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol:240]: By default `_isOracleDrivenLaneAllowed(_token, _from, _receiver)` always return true that mean oracle will automatically `executeSignature` that mint/unlock NFT on the foreign so oracle will pay a transaction fee. If we set `false` oracle will skip that so user can claim that

Comment (kazkondo) [reply contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol:240]: OK. Please remain this code as commented out code and put the comment to here

Comment (cuonghx-dev) [reply contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol:240]: I've already done it

**Pull request merged**